### PR TITLE
Supplement empty filter user inputs with sample 1

### PR DESCRIPTION
**Pull request by bugraderre**
Closes: #2

### CODE REVIEW

File: pdchaosazure/common/compute/command.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def run(resource_group: str, compute: dict, timeout: int, parameters: dict,
             .format(compute.get('type'))
         raise InterruptExecution(msg)
 
-    result = poller.result(timeout)  # Blocking till executed
+    result = poller.result()  # Blocking till executed
```
[the remark this answers is not in the document]
Comment (bugraderre) [reply pdchaosazure/common/compute/command.py:61]: Good one. Forgot to roll it back on my machine.

File: pdchaosazure/common/resources/query.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    return result
+
+
+def __prepare(resource_type: str, user_query: str) -> str:
+    result = ["Resources", "where type=~'{}'".format(resource_type)]
+
+    if user_query:
+        result.append(user_query)
+    else:
+        result.append("sample 1")
```
[the remark this answers is not in the document]
Comment (bugraderre) [reply pdchaosazure/common/resources/query.py:26]: Exact

File: pdchaosazure/common/resources/graph.py [hunk 3]
```diff
@@ -2,24 +2,23 @@
...
 from chaoslib.exceptions import InterruptExecution
 from chaoslib.types import Secrets, Configuration
 
 from pdchaosazure import init_resource_graph_client
-from pdchaosazure.common.config import load_configuration
+from pdchaosazure.common.resources import query
 
 
-def fetch_resources(input_query: str, resource_type: str,
+def fetch_resources(user_query: str, resource_type: str,
```
[the remark this answers is not in the document]
Comment (bugraderre) [reply pdchaosazure/common/resources/graph.py:13]: It is a query with which we filter. I have the feeling we should call it query in the user input as well. Everywhere.

File: pdchaosazure/common/compute/command.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkaszub) [reply pdchaosazure/common/compute/command.py:61]: so this change is not part of this PR right? Please remove it than and let's take care of timeout in other PR

File: pdchaosazure/common/resources/graph.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (mkaszub) [reply pdchaosazure/common/resources/graph.py:13]: ok we can think of it, when we will be doing story with simplified quering/filtering mechanism

**Pull request merged**